src/components/cards/TradeCard/TradeCard.vue

```diff
@@ -192,22 +195,21 @@ export default defineComponent({
       initSor,
       handleAmountChange,
       priceImpact,
-      exactIn,
       sorReturn,
       latestTxHash,
       pools,
       fetchPools,
       poolsLoading
-    } = useSor(
-      tokenInAddress,
-      tokenInAmount,
-      tokenOutAddress,
-      tokenOutAmount,
+    } = useSor({
+      exactIn,
+      tokenInAddressInput: tokenInAddress,
+      tokenInAmountInput: tokenInAmount,
+      tokenOutAddressInput: tokenOutAddress,
+      tokenOutAmountInput: tokenOutAmount,
       tokens,
-      allowanceState,
       isWrap,
       isUnwrap
-    );
+    });
```

Comment:
these are minimal changes to allow passing an object since there were too many params.